packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -199,6 +204,11 @@ class CallContext {
       call = undefined;
     }
 
+    /* @conditional-compile-remove(call-transfer) */
+    const transferCall = call?.transferFeature?.acceptedTransferCallId
+      ? clientState.calls[call.transferFeature.acceptedTransferCallId]
```

Comment:
By storing this here we are duplicating the state. I'm not against this, but it may be cleaner to just store the `acceptedTransferCallId` and the perform the lookup when it's used

Reply:
Actually, I guess the adapter doesn't expose this, never mind!